tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -1504,6 +1502,33 @@ def test_limit_without_orderby_excess_groups_pruned(self):
             ],
             entity="metrics_sets",
         )
+        response = self.get_success_response(
+            self.organization.slug,
+            field=[
+                SessionMetricKey.ERRORED.value,
+                "sum(sentry.sessions.session)",
+            ],
+            statsPeriod="1h",
+            interval="1h",
+            groupBy="tag1",
+            per_page=3,
+        )
+
+        groups = response.data["groups"]
+        assert len(groups) == 3
+
+    def test_limit_without_orderby_excess_groups_pruned_perfromance(self):
```

Comment:
I am afraid the tests are still incorrect because now they are no longer testing pruning behavior